numpy/ma/core.py

```diff
@@ -7088,6 +7123,24 @@ def putmask(a, mask, values):  # , mode='raise'):
     Using a masked array as `values` will **not** transform a `ndarray` into
     a `MaskedArray`.
 
+    Examples
+    --------
+    >>> x = np.arange(6).reshape(2, 3)
+    >>> np.putmask(x, x>2, x**2)
+    >>> x
+    array([[ 0,  1,  2],
+       [ 9, 16, 25]])
```

Comment:
I suspect this should align properly:
```suggestion
           [ 9, 16, 25]])
```
Should do it (I think).  Although it probably doesn't trip the test suite at least.